bin/post_compile

```diff
@@ -0,0 +1,6 @@
+echo SFDO_METACI_DEPLOY=${SFDO_METACI_DEPLOY}
+if [[ -z "${SFDO_METACI_DEPLOY}" ]]; then
+    echo Installing from sfdo-metaci.txt
+    pip3 install -r requirements/sfdo-metaci.txt > /dev/null 2>&1 || echo Pip install failed. Error suppressed for security reasons
```

Comment:
Does it include the github username & access token in the log otherwise? Since this is suppressed, did you validate that snowfakery was successfully installed in some other way?

Reply:
> Does it include the github username & access token in the log otherwise?

Yes.

> did you validate that snowfakery was successfully installed in some other way?

When we get to staging I will log into a dyno and verify.

Reply:
Okay. I wonder if we need to add `#egg=snowfakery` to the end of the git URL so that pip knows what package to install, but I guess we'll find out.